features/events/CreateEventScreen.js

```diff
@@ -45,7 +46,15 @@ class CreateEventScreen extends Component {
   }
 
   _create = async () => {
-    const {goBack, state} = this.props.navigation;
+    const {navigate, state} = this.props.navigation;
+    const {currentUser} = this.props.screenProps.auth;
+    const participations = this.state.invitees.map(i => ({
+      role: 'MEMBER',
+      userId: i,
+    }));
+    if (currentUser) {
+      participations.push({role: 'OWNER', userId: currentUser.id});

```

Comment:
I tried to auto-associate the current user as the "owner" of the event on Graph.cool but their hooks are pretty limited. This was much easier.

Reply:
👍